pkg/server/claims_test.go

```diff
@@ -80,14 +83,43 @@ func TestTokenClaimString(t *testing.T) {
 			expErrMsg: "",
 		},
 	}
+	boolCases := []TestCase[bool]{
```

Comment:
I'm not sure why we need to break these apart. It's normal to have hundreds or thousands of table-driven tests (https://github.com/sethvargo/go-envconfig/blob/main/envconfig_test.go). If you need/want to namespace things, you can use slashes in the names.

Reply:
I only broke them apart because I didn't know how to have a slice of generic objects with different instantiated types. I didn't realize I could use `any`, thanks!

Reply:
Refactored the test to have all the cases defined in the same place with an inline struct. Unfortunately I still had to have a helper function which calls `tokenClaim[T]` because Go couldn't infer the type otherwise.